configurator/src/main.rs

```diff
@@ -131,6 +131,17 @@ struct BitcoinChannelConfig {
     time_lock_delta: usize,
 }
 
+#[derive(serde::Deserialize)]
+#[serde(rename_all = "kebab-case")]
+struct SweeperConfig {
+    sweeper_maxfeerate: u64,
+    sweeper_nodeadlineconftarget: usize,
+    sweeper_budget_tolocalratio: f64,
+    sweeper_budget_anchorcpfpratio: f64,
+    sweeper_budget_deadlinehtlcratio: f64,
+    sweeper_budget_nodeadlinehtlcratio: f64,
+}
```

Comment:
I think you also want to include `Option<usize>` values for the different fee caps